components/Legals/Legals.tsx

```diff
@@ -1,3 +1,52 @@
-import { UnderConstruction } from '../common/UnderConstruction';
-
-export const Legals: React.FC = () => <UnderConstruction />;
+export const Legals: React.FC = () => (
+    <div className="legals">
+        <h1>Mentions légales</h1>
+        <h2 id="editeur-de-la-plateforme">Editeur de la plateforme</h2>
+        <p>
+            La Plateforme est éditée par la{' '}
+            <a href="https://incubateur.social.gouv.fr/">
+                Fabrique des Ministères sociaux
+            </a>{' '}
+            située :
+        </p>
+        <p>Tour Mirabeau</p>
+        <p>39-43 Quai André Citroën</p>
+        <p>75015 PARIS</p>
+        <p>Tél : 01 40 56 60 00</p>
+        <h2 id="directeur-de-la-publication">Directeur de la publication</h2>
+        <p>Hélène Brisset, Directrice du Numérique</p>
```

Comment:
```suggestion
        <p>Anne Jeanjean, Directrice du Numérique</p>
```
cf: https://www.gouvernement.fr/conseil-des-ministres/2022-04-20#mesures-d-ordre-individuel